### ISSUE TITLE
Expose the raw body on the mock server call log

### ISSUE THREAD

**Issue by mustela** [a85/POSTMan-Chrome-Extension#8334]
When using the mock server, it would be super useful to have the log exposing the raw data, like the body, headers, and not wrapping them into the UI. 
<img width="956" alt="Screenshot 2020-04-15 at 10 52 16" src="https://user-images.githubusercontent.com/325222/79318639-8751b900-7f07-11ea-95d9-bf9b43e625bd.png">

Although it's a nice ui, it's hard to use when you are comparing different calls, and trying to understand why calls are not being matched, etc. 

Thanks!

**Comment by vvs11** [comment]
@mustela Thanks for the feedback. We will keep a note of this and work upon improving the experience

**Comment by DanDepot** [comment]
I am also in need of the mock server log showing the request and response body as raw text in the same way you can see the request and response body on a normal postman request as raw text.

Is there any status on this change?

**Comment by vvs11** [comment]
@DanDepot @mustela Could you elaborate on your use-case here? Also, would it be possible to get on a call and understand more about how you use postman mocks. This would help us build better features in future.

**Comment by pinko-fowle** [comment]
@vvs11 We're trying to use some of the data we get from the mock endpoint in our code, but we can't effectively use the data we get on the mock server because the UI keeps us from being able to actually look at the real data. :(

**Comment by ebrown5-captioncall** [comment]
I have a similar situation - I've created calls with examples and tested them using Postman to get successful responses.

When I call from my app, I cannot get the example to be found (I get "no matching requests" in the log)

I've turned off header matching in the mock server configuration, so copying headers should not be the answer.

I copied and pasted the body from the "failed" request into the example and still have no matches.

I will be creating a related feature request: provide an affordance to create a request with examples from "no matching request" log entries.  This would allow me to skip the process of debugging why my mocks are failing and move forward with testing my products.

**Comment by Pulseweaver** [comment]
I get "No matching requests" when making a call to the postman mock service. I suspect that there need to be a expected request registered in the mock but I don't really know how it looks like. So if postman could just let me see and copy the incoming request it would be easier to setup the mock service and avoid this warning.

**Comment by ebrown5-captioncall** [comment]
Yeah, it's a bit annoying to not have any idea what's causing the issue.

I made a copy of the "get mock server logs" example and use that to see the full details and it's helped.

Still, it'd be great to have the ability to make an example out of a failed request.  Looking at the data structures, it's not super simple to do, but it's far from impossible.  And I have no idea how to integrate that into the postman UI.

The specific issue I had when I first posted my suggestion was the app that was making the call was using lower-case for the "Content-Type" header.  When I changed my example to match, it started working.
________________________________
